client/rpcserver/types.go

```diff
@@ -181,10 +181,12 @@ func parseNewWalletArgs(params *RawParams) (*newWalletForm, error) {
 		AppPass:    params.PWArgs[0],
 		WalletPass: params.PWArgs[1],
 		AssetID:    uint32(assetID),
-		Account:    params.Args[1],
```

Comment:
For the rpcserver, account is still needed because it will not be in the config file. It can be taken as an argument here and inserted after parsing.

Reply:
The problem is that it's not a universal argument. It's already misnamed for btc, where it should be called a wallet, not an account. I also don't know that it will be required for future assets. So it makes sense to make it an asset-specific configuration option, imo. 

This change does mean that in RPC server you can no longer scan the dcrwallet configuration file directly, so that sucks. I've created a separate config file for myself and added the entry for `account`, but I understand that that's not great UX. In the app, it's not a problem, since you can still edit the settings after reading from file. I wonder if we can find an analogue to that in the RPC server. Either a) a command to modify or add a single setting, or b) a way to pass additional args when creating the wallet. I think **a** is easier and more versatile.

Thoughts?

Reply:
I'm not sure what a would look like. Would it involve breaking up the rpc into setting the config and calling create later, with less params?

b sounds ok to me. For example, an optional map struct that you could send like {"account": "default", "wallet": "gamma", ...} and have the key/value pair added the config.

Another option is to just make separate commands. `newdcrwallet` and `newbtclikewallet`.

Reply:
Option (a) is tricky since I don't think creating the wallet and then setting the correct account would be a good workflow and could lead to bugs on the backend.  Perhaps I'm not clear on the suggestion either.

(b) actually sounds appealing to me too, with an extra optional arg that is a json object just like @JoeGruffins suggested.  I'm less excited about different new wallet commands (like `newdcrwallet` and `newbtclikewallet`) because it's not generalized and could get out of hand over time.

Reply:
> I've created a separate config file for myself and added the entry for `account`, but I understand that that's not great UX

To support this without risking wallet config file setting conflicts, we could have this be `dexaccount` or `tradingaccount` perhaps.

Reply:
It's the unknown flag error that'll getcha. But I think you've convinced me on the json object arg. Should I do it here?

Reply:
I think this also needs to be changed to `1` as the parameter has moved.

https://github.com/decred/dcrdex/blob/36423893bf09d3e3b4d4b815eafca98f9a1f5d38/client/cmd/dexcctl/main.go#L71